### ISSUE TITLE
Plugin?

### ISSUE THREAD

**Issue by mpsparrow** [prionkor/limit-login-sessions#1]
Have you considered turning this into a plugin for the WordPress directory? I think it would make a great plugin.

**Comment by prionkor** [comment]
I wasn't sure if it will be useful but I can surely make it a plugin if you think it may be! :)

**Comment by zebrastribe** [comment]
Agreed it would be nice if it was a plugin. Also there could be a option include or exclude wordpress roles so that it not all the users that is a limit on...
